src/apps/util/config.clj

```diff
@@ -123,6 +123,16 @@
   [props config-valid configs]
   "apps.tools.private.memory-limit" (* 16 1024 1024 1024)) ;; 16GB
 
+(cc/defprop-optint default-cpu-limit
+  "The CPU limit, in cores (matching the database's data type), to be used as the default resource limit when one is not set in a tool definition."
+  [props config-valid configs]
+  "apps.tools.default.cpu-limit" 4)
```

Comment:
Unsure if this is too low, but I just copied it from the app-exposer hardcoded default for now.

Reply:
That seems like a reasonable default. We can always tweak it in the configuration files if necessary.